namespace java helloworld
namespace py helloworld

service HelloWorld
{
  string ping(1:string name),
}